<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopRightButtons</class>
 <widget class="QWidget" name="TopRightButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>35</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="control::NavPushButton" name="notificationButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/NavNotificationNormal.png</normaloff>:/img/NavNotificationNormal.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="control::NavPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/NavSettings.png</normaloff>:/img/NavSettings.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="control::NavPushButton" name="accountButton">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/img/NavAccount.png</normaloff>:/img/NavAccount.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::NavPushButton</class>
   <extends>QToolButton</extends>
   <header>control/NavControls.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
